automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -146,10 +148,21 @@ def satellite6_capsule_upgrade(admin_password=None):
         print('Wrong Capsule Version Provided to upgrade to. '
               'Provide one of 6.1, 6.2')
         sys.exit(1)
+    if not os.environ.get('CAPSULE_SUBSCRIPTION') and not os.environ.get(
+            'CAPSULE_AK'):
+        print('One of Env. Var CAPSULE_SUBSCRIPTION or CAPSULE_AK should be'
+              'defined!')
+        sys.exit(1)
     if admin_password is None:
         admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
     # Setting Capsule61 Repos
     major_ver = distro_info()[1]
+    # Re-register Capsule for 6.2¬
```

Comment:
Is this a workaround? If yes, please incorporate the Bug ID to track where this comes from.